src/Core/Manifest/VersionProvider.php

```diff
@@ -40,15 +49,85 @@ class VersionProvider
     public function getVersion()
     {
         $modules = $this->getModules();
+        $this->updateModulesWithLockFileRecipes($modules);
         $lockModules = $this->getModuleVersionFromComposer(array_keys($modules));
-        $output = [];
+
+        $arr = [];
+        $unknownVersion = _t(__CLASS__ . '.VERSIONUNKNOWN', 'Unknown');
         foreach ($modules as $module => $title) {
-            $version = isset($lockModules[$module])
-                ? $lockModules[$module]
-                : _t(__CLASS__ . '.VERSIONUNKNOWN', 'Unknown');
-            $output[] = $title . ': ' . $version;
+            $version = $lockModules[$module] ?? $unknownVersion;
+            $arr[$module] = "$title: $version";
+        }
+        $this->filterPriorityModules($arr);
+        return implode(', ', array_values($arr));
+    }
+
+    /**
+     * Update modules to include two silverstripe recipes, if they exist in the composer.lock file
+     *
+     * Hard-coding here instead of adding to respective module _config/config.yml
+     * because recipes are not loaded, instead contents of modules app/_config/*.yml
+     * are copied to the project app/_config folder, and it would be annoying for usres to have
+     * 2x recipe-config.yml files copied into their projects _config folder just for version labels
+     *
+     * @param $modules
+     * @return void
+     */
+    private function updateModulesWithLockFileRecipes(&$modules): void
+    {
+        $data = [
+            'silverstripe/recipe-core' => 'Core Recipe',
+            'silverstripe/recipe-cms' => 'CMS Recipe',
```

Comment:
Are we concerned that these titles are not localisable?

Reply:
Probably not, the config elsewhere for the normal module labels is not localised either. 
https://github.com/silverstripe/silverstripe-cms/blob/4/_config/version.yml

I don't think it makes sense to make this bit localisable and not the other